CodeXL/Components/PowerProfiling/Backend/AMDTPowerProfilingDrivers/inc/AMDTRawDataFileHeader.h

```diff
@@ -83,7 +83,7 @@ typedef enum
     RAW_UNKNOWN_ERROR = 3,
 } RAW_Status;
 
-static const wchar_t g_RawFileMagicStr[] = L"_RAWHDR_";
+static const wchar_t g_RawFileMagicStr[] = {'', 'R', 'A', 'W', 'H', 'D', 'R', ''};
```

Comment:
Should be:

`static const wchar_t g_RawFileMagicStr[] = {'_', 'R', 'A', 'W', 'H', 'D', 'R', '_'};`